pkg/3scale/amp/component/system.go

```diff
@@ -269,16 +280,6 @@ func (system *System) buildSystemBaseEnv() []v1.EnvVar {
 	systemBackendInternalAPIPass := envVarFromSecret("CONFIG_INTERNAL_API_PASSWORD", "backend-internal-api", "password")
 	result = append(result, systemBackendInternalAPIUser, systemBackendInternalAPIPass)
 
-	if system.Options.s3FileStorageOptions != nil {
-		result = append(result,
```

Comment:
We remove all the envvars as they are not used anymore.
At this moment we should decide how system will be aware of in what mode is in: S3 or PVC. We notified that to System by setting the "FILE_UPLOAD_STORAGE" environment variable to "s3" but not sure whether this approach is the one we want. Maybe System could just check if the configuration file exists? If the secret exists or the volume is mounted or a directory is created?

Reply:
As I understood from the meeting with system guys, `FILE_UPLOAD_STORAGE` is still there to notify type of storage

Reply:
It is still there. I was saying this to evaluate/discuss whether another approach would be better than using the environment variable

Reply:
@thomasmaas  who is going to work on this task in system? we would like to know his/her opinion about this